libs/langchain/langchain/utilities/graphql.py

```diff
@@ -7,14 +7,17 @@
 class GraphQLAPIWrapper(BaseModel):
     """Wrapper around GraphQL API.
 
-    To use, you should have the ``gql`` python package installed.
+    To use, you should have the ``gql`` and ``graphql`` python packages installed.
     This wrapper will use the GraphQL API to conduct queries.
     """
 
     custom_headers: Optional[Dict[str, str]] = None
+    disable_schema_prompt: bool = False
```

Comment:
Would you mind renaming this parameter to better indicate it's role? 

Some potential names could be `auto_fetch_schema` or `fetch_schema_if_missing` or something else that's similar. 

The name would ideally help communicate to developers that this parameter potentially controls an expensive initialization operation (a network request to fetch schema). 

The wrapper should also not be aware of the fact that the schema is being fetched to populate a prompt by the tool that uses the wrapper.

Reply:
renaming to auto_fetch_schema and making that change for who controls schema fetching

Reply:
Just made that change, updated the pr